src/qt/rpcconsole.cpp

```diff
@@ -463,13 +463,15 @@ void RPCConsole::clear()
 
     // Set default style sheet
     ui->messagesWidget->document()->setDefaultStyleSheet(
+        QString(
                 "table { }"
                 "td.time { color: #808080; padding-top: 3px; } "
+                "td.message { font-family: monospace; font-size: %1pt; } "
                 "td.cmd-request { color: #006060; } "
                 "td.cmd-error { color: red; } "
                 "b { color: #006060; } "
-                );
-
+            ).arg(QFont("Monospace").pointSize() * 4 / 5) // 0.8 em
```

Comment:
Thanks, I have alrady replaced the code with a version that should work on all platforms (and maybe even HiDPI).